HHS/Head-Start-TTADP
# [Prod] Update reason, user administration improvements, fix CSV export bug, final bits of communication log

## Description of change
- Change reason "Planning/Coordination (also TTA Plan Agreement)" to "Planning/Coordination"
- Rearrange user administration panel: add "all" so that all regions are selected, place permissions into sections, condense permission names, remove training_reports flag
- Fix bug with other entity CSV export
- Finish communication log: add RTTAPA updates table, export communication log buttons, misc fixes to the UI

## How to test
- Confirm the reason appears correctly in the UI
- Confirm the changes to the user administration section make sense and meet acceptance criteria
- You can export other-entity CSV
- View the communication log with the feature flag and note that it looks correct

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2141
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1991
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2352
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2247
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2246
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2364
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2143

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
